core/src/main/java/fr/sncf/osrd/stdcm/graph/STDCMPathfinding.java

```diff
@@ -112,18 +127,17 @@ private static double totalDistanceUntilEdgeLocation(
         var envelope = range.edge().envelope();
```

Comment:
can this function be renamed so that its name does not imply that it returns a distance? it returns a weight